task7.py

```diff
@@ -0,0 +1,21 @@
+# 7. Задание: Даны два натуральных числа.
+# Вычислите их наибольший общий делитель
+# при помощи алгоритма Евклида
+a = int(input())
+b = int(input())
+# запускам цикл и меньшее число
+# делим на большее,
+# результат сохраняем в этой же
+# переменной
+while a != 0 and b != 0:
+    if a > b:
+        a %= b
+    else:
+        b %= a
+# т.к. заранее не известно в какой
+# переменной сохранится результат
+# складываем переменные, так как
+# во второй все равно в конце
+# цилкба будет 0
+most_overall_divider = a + b
```

Comment:
`gcf = a if a else b. `
you write a cheat.

Reply:
What exactly is a cheat? The code is working.